pkg/registry/policy/poddisruptionbudget/strategy.go

```diff
@@ -83,7 +83,13 @@ func (podDisruptionBudgetStrategy) AllowCreateOnUpdate() bool {
 
 // ValidateUpdate is the default update validation for an end user.
 func (podDisruptionBudgetStrategy) ValidateUpdate(ctx context.Context, obj, old runtime.Object) field.ErrorList {
-	return validation.ValidatePodDisruptionBudget(obj.(*policy.PodDisruptionBudget))
```

Comment:
@liggitt Just out of curiosity, it seems the original `ValidateUpdate` doesn't take the existing object into account, why do we need to consider the existing object when tightening the validation if our purpose is not to accept invalid PDB anymore?

Reply:
Whenever we tighten validation, it *always* has to be done with regard for an existing persisted object so we never block a no-op update of an existing object.